fplll/pruner/pruner.h

```diff
@@ -16,7 +13,11 @@
    You should have received a copy of the GNU Lesser General Public License
    along with fplll. If not, see <http://www.gnu.org/licenses/>. */
 
+#ifndef FPLLL_PRUNER_H
+#define FPLLL_PRUNER_H
+
 #include "defs.h"
+#include "lll.h"
```

Comment:
Those includes should be for "fplll/defs.h" and "fplll/lll.h", otherwise third party code like FPyLLL  won't build.

Reply:
Thanks. I changed them. Let me know if there are other header problems.